TidepoolOnboardingKitUI/View Models/PrescriptionReviewViewModel.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  PrescriptionCodeEntryViewModel.swift
```

Comment:
Ditto here.  In fact, I'm going to assume all of the code from TidepoolService was just copy/pasted here.